docs/3.0.0-beta.x/community/contributing-guide.md

```diff
@@ -0,0 +1,75 @@
+# Open Development & Community Driven
+
+Strapi is an open-source project administered by [the Strapi team](https://strapi.io/company). We appreciate your interest and efforts in contributing to Strapi.
+
+There are many ways that you can contribute to Strapi. Contributions to Strapi and the greater Strapi Community can take the form of answering questions in one of the [many Strapi channels](/3.0.0-beta.x/community/contributing-guide.html#get-in-touch), writing articles, creating videos and otherwise publishing self-generated content.
+
+The two additional ways to contribute to Strapi include contributing **Documentation** and **Code**:
+
+- [Contribute in the form of Code Contributions](/3.0.0-beta.x/community/contribute-with-code.md)
+- [Contribute in the form of Documentation, Tutorials, or Articles](/3.0.0-beta.x/community/contribute-with-docs.md).
+
+Some people choose to support Strapi by [purchasing Swag from our Store](https://strapi.io/shop), (and then proudly wearing it!).
+
+Any way you choose to contribute to Strapi is welcomed and appreciated.
+
+## Open-Source & MIT Licensed
+
+Strapi is open-source under the [MIT license](https://github.com/strapi/strapi/blob/master/LICENSE.md). All work done is available on [GitHub](https://github.com/strapi/strapi).
+
+The core team, as well as any contributors, send pull requests which then go through the same validation process.
+
+## Code of Conduct
+
+The [Code of Conduct](/3.0.0-beta.x/community/code-of-conduct.md) governs this project and everyone participating in it. By participating, you are expected to uphold this code. Please read the [full text](/3.0.0-beta.x/community/code-of-conduct.md) so that you can read which actions may or may not be tolerated.
+
+## Feature Requests
+
+Feature Requests by the community are highly encouraged. Please feel free to submit a [feature request](https://portal.productboard.com/strapi) or to upvote 👍 [an existing feature request](https://portal.productboard.com/strapi) in the ProductBoard.
```

Comment:
Please feel free to submit a [feature request](https://portal.productboard.com/strapi) or upvote 👍

Reply:
Yes. that reads better. thank you.